collect/cache/traceSentCache.go

```diff
@@ -0,0 +1,24 @@
+package cache
+
+import (
+	"github.com/honeycombio/refinery/types"
+)
+
+type TraceSentRecord interface {
+	// Kept returns whether the trace was kept (sampled and sent to honeycomb) or dropped.
+	Kept() bool
```

Comment:
As we're introducing an interface, can we consolidate the wording to match the usage / sampler? eg we use kept for the record, and should send from the sampler. Would `Sent` be clearer?

Reply:
I did consider that, but there's code in refinery that uses "send" to mean, effectively, "do something with this" and the result is either keep or drop. So I think it's clearer to see this from the cache's point of view.